src/FNA3D_Driver_OpenGL.c

```diff
@@ -349,8 +349,8 @@ static int32_t XNAToGL_TextureInternalFormat[] =
 {
 	GL_RGBA8,				/* SurfaceFormat.Color */
 	GL_RGB8,				/* SurfaceFormat.Bgr565 */
-	GL_RGB5_A1,				/* SurfaceFormat.Bgra5551 */
-	GL_RGBA4,				/* SurfaceFormat.Bgra4444 */
+	GL_BGRA,				/* SurfaceFormat.Bgra5551 */
```

Comment:
These two I for sure need XNA comparison tests. I used unit tests for most of these subbyte formats years ago and haven't touched them since.